### DISCUSSION TITLE
Side-by-Side diff

### DISCUSSION THREAD

**Discussion by Craige** [community/community#10867] [Mobile] [answered]
Would love to have the option for side-by-side diffs on PR where screen size is abundant enough. iPad in landscape has plenty of real estate, but all I can see is a a standard patch style diff. Makes pull requests on the app a no-go for anything but the most basic changes.

**Comment by amrehman** [comment, marked as answer]
Hi @Craige !
Thanks a lot for this feedback!

I am going to add your request to our internal notes where we are tracking side by side diffs, and we will discuss this as a team and try to get this on your device as soon as possible.

Thanks for using our app and please keep the feedback coming!

**Comment by ssbarnea** [comment]
TBH, I am still trying to find any diff option on iphone, very annoying to not be able to review PRs on mobile.
